static-site/content/blog/2024-03-27-annual-update-march-2024.md

```diff
@@ -0,0 +1,53 @@
+
+Building on the popularity of **Nextclade** in the early pandemic, much of the activities of the Basel team has been focused on developing a more general and robust version. In January 2024, we released [version 3](https://github.com/nextstrain/nextclade/releases/tag/3.0.0) that supports (a) complex annotations, including splicing, slippage, and circular genomes, (b) viruses with up to 30% sequence divergence, (c) simple tree building instead of just phylogenetic placement, (d) automatic suggestion of suitable datasets and (e) datasets contributed and maintained by the community. Nextclade currently supports SARS-CoV-2, seasonal influenza viruses, RSV, MPXV and PRRSV-2 (via community).
+
+Development on **nextstrain.org** focused on sharing and surfacing datasets. We have supported ["community" datasets](https://docs.nextstrain.org/en/latest/guides/share/community-builds.html) since 2018 through the ability to display phylogenetic analyses deposited on GitHub. More recently, the Seattle team has implemented an integrated solution called ["Nextstrain Groups" where users can maintain their own public or private analyses](https://docs.nextstrain.org/en/latest/learn/groups/index.html). There are currently 41 user groups, including public health authorities, with groups datasets managed via the CLI. The same credentials to login via `nextstrain login` are used to login to nextstrain.org via browser. We've recently made improvements to AWS infrastructure and permissions to streamline inclusion of new groups, but this still requires individuals to reach out to us via email. We've provided a whitelisted version of Nextstrain Groups for the US CDC to use internally. In addition, we've implemented new functionality to surface ["snapshots" of historical datasets](https://docs.nextstrain.org/en/latest/guides/snapshots.html) on nextstrain.org.
+
+Development of **Auspice tree visualization** has focused on inclusion of a ["measurements panel"](https://www.frontiersin.org/articles/10.3389/fbinf.2023.1069487/full) intended for displaying titer or DMS data and on improvements to the "entropy panel" to support more complex genome architectures.
+
+In terms of **training and technical support** we've provided standalone installers for Linux, Mac, and Windows (allowing one-command installation) and new managed runtimes for Conda and Singularity, centralized documentation and tutorials at [docs.nextstrain.org](https://docs.nextstrain.org), hosted weekly drop in office hours via Zoom, responded to user emails to hello@nextstrain.org and responded to posts to discussion board [discussion.nextstrain.org](https://discussion.nextstrain.org). We've also led hands-on training sessions at VEME in Aug 2022 and Aug 2023 and hosted intern Alhaji Olono from ACEGID in summer 2023 to work on Nextstrain build for African Swine Fever Virus.
+
```

Comment:
```diff
- to work on Nextstrain build for African Swine Fever Virus
+ to work on a Nextstrain build for African Swine Fever Virus
```

Reply:
Resolved in 47b6ab322fca3125cdee07a6b1dcc7fc07c47b0a